lib/install.ps1

```diff
@@ -355,7 +355,17 @@ function args($config, $dir) {
 function run($exe, $arg, $msg, $continue_exit_codes) {
     if($msg) { write-host $msg -nonewline }
     try {
-        $proc = start-process $exe -wait -ea stop -passthru -arg $arg
+        #Check if arguments were provided in the manifest, and pass them to the installer if present
+        if ($arg)
```

Comment:
I think this will work....

```
    # allow for null $arg
    $params = @{ }
    if($arg) {
        $params.arg = $arg;
    }
```

and then instead of the if/else:
$proc = start-process $exe -wait -ea stop -passthru @params

Minor detail, but avoids repetition.